taskt/Core/Automation/Commands/Data Commands/FormatDataCommand.cs

```diff
@@ -1,76 +1,74 @@
-        [Attributes.PropertyAttributes.SampleUsage("MM/dd/yy, hh:mm, C2, D2, etc.")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Output Data Format")]
+        [InputSpecification("Specify the output data format.")]
+        [SampleUsage("MM/dd/yy, hh:mm:ss || C2 || D2")]
+        [Remarks("")]
         public string v_ToStringFormat { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select the variable to receive output")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_applyToVariableName { get; set; }
+        [PropertyDescription("Output User Variable")]
+        [InputSpecification("Select or provide a variable from the variable list.")]
+        [SampleUsage("vUserVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required" +
+                 " to pre-define your variables; however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
 
         public FormatDataCommand()
         {
-            this.CommandName = "FormatDataCommand";
-            this.SelectionName = "Format Data";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
-
-            this.v_InputValue = "{DateTime.Now}";
-            this.v_FormatType = "Date";
-            this.v_ToStringFormat = "MM/dd/yyyy";
-            
+            CommandName = "FormatDataCommand";
+            SelectionName = "Format Data";
+            CommandEnabled = true;
+            CustomRendering = true;
+
+            //v_InputData = "{DateTime.Now}";
+            //v_FormatType = "Date";
+            //v_ToStringFormat = "MM/dd/yyyy";
         }
 
         public override void RunCommand(object sender)
         {
             //get variablized string
-            var variableString = v_InputValue.ConvertToUserVariable(sender);
+            var variableString = v_InputData.ConvertToUserVariable(sender);
 
             //get formatting
             var formatting = v_ToStringFormat.ConvertToUserVariable(sender);
-
-            var variableName = v_applyToVariableName.ConvertToUserVariable(sender);
-
+            
+            //get output variable
+            var variableName = v_OutputUserVariableName.ConvertToUserVariable(sender);
```

Comment:
this variable is not used. Please delete this line

Reply:
done